tool/src/test/resources/org/datacommons/tool/genmcf/resolution/output/report.json

```diff
@@ -117,5 +118,99 @@
     },
     "userMessage": "Found a local ref to an unresolvable node :: ref: 'l:SVId', property: 'variableMeasured', node: 'UnresSVObsId'",
     "counterKey": "Resolution_ReferenceToFailedNode_variableMeasured"
+  }],
+  "statsCheckSummary": [{
+    "placeDcid": "wikidataId/Q1437",
+    "statVarDcid": "CumulativeCount_MedicalTest_ConditionCOVID_19_Positive",
+    "measurementMethod": "",
+    "observationPeriod": "",
+    "scalingFactor": "",
+    "unit": "",
+    "seriesLargestPercentDiff": {
+      "percentDifference": 0.0
+    }
```

Comment:
This seems spammy... should we skip when percentDiff is 0?

Reply:
done